tests/PageObjects/RemoteMic/RemoteMicMainPage.ts

```diff
@@ -15,7 +16,31 @@ export class RemoteMicMainPagePO {
     await this.playerNameInput.fill(name);
   }
 
+  public async expectPlayerNameToBe(playerName: string) {
+    await expect(this.playerNameInput).toHaveValue(playerName);
+  }
+
   public async goToSongList() {
     await this.page.getByTestId('menu-song-list').click();
   }
+
+  public async goToSettings() {
+    await this.page.getByTestId('menu-settings').click();
+  }
+
+  public async expectPlayerToBeAssigned(micColor: string) {
+    if (micColor === 'blue') {
+      await expect(this.page.getByTestId('indicator')).toHaveAttribute('data-player-number', '0');
```

Comment:
you can declare an object mapping color to the number, and use it instead:

```ts
const colorToNumberMap = {
   blue: 0, // etc.
};
//[...]
await expect(this.page.getByTestId('indicator')).toHaveAttribute('data-player-number', colorToNumberMap[micColor]);
```
This will simplify the code and let you not repeat `expect(this.page.getByTestId)` for each color

Reply:
Great, thank you! :)